src/ruby/symengine.gemspec

```diff
@@ -0,0 +1,16 @@
+Gem::Specification.new do |gem|
+	gem.name = 'symengine'
+	gem.version = '0.0.0.pre'
+	gem.summary = "Ruby interface for SymEngine, a fast C++ library for symbolic manipulation"
+	gem.description = "This gem provides a Ruby interface for SymEngine, a fast C++ library for symbolic manipulation, so that the you can use the features of SymEngine in ruby."
+	gem.authors = ['Abinash Meher']
+	gem.email = ["abinashdakshana999@gmail.com"]
+	gem.homepage = 'https://github.com/sympy/symengine'
+	gem.files = Dir["lib/**/*", "bin/*", "../../LICENSE", "*.md"]
```

Comment:
Again, my fault. I mistook that variable for the files that get copied to the gem after installation. That's why I gave it the relative path to the symengine LICENSE file. Is there any other way, where I can avoid the file duplication?